desc/compute/_curve.py

```diff
@@ -635,6 +635,88 @@ def _x_sss_FourierXYZCurve(params, transforms, profiles, data, **kwargs):
     return data
 
 
+def _splinexyz_helper(f, p_knots, transforms, xq, kwargs, derivative):
+    method = kwargs.get("method", "cubic")
```

Comment:
some comments to explain what this is doing (I know it is private but it is long) would be good